.github/workflows/ci.yml

```
@@ -1,13 +1,31 @@
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '18'
+          cache: 'npm'
+      - name: Install dependencies
+        run: npm install
+      - name: Lint markdown files
+        run: npm run lint
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)